akka-docs/src/main/paradox/stream/operators/Source-or-Flow/take.md

```diff
@@ -16,6 +16,15 @@ Pass `n` incoming elements downstream and then complete
 
```

Comment:
we had an idea of how to include the signature for Java, see RetryFlow withBackoff.md

Reply:
Hmm, that's kind of weird though, why repeat the signature in a test instead of just `// #name` tagging the actual signature?

Reply:
Ah, the actual method signature is in Scala, ok.